### DISCUSSION TITLE
Error while opening installer on linux

### DISCUSSION THREAD

**Discussion by ghost** [open-watcom/open-watcom-v2#1126] [Q&A]
I downloaded the installer using  wget https://github.com/open-watcom/open-watcom-v2/releases/download/Current-build/open-watcom-2_0-c-linux-x64  and when I open the installer with  sudo ./open-watcom-2_0-c-linux-x64  I get a malloc corrupted top size error and the terminal just goes crazy. How do I fix it?

**Comment by jmalak** [comment]
Please, give more info about your system.
Linux distribution and version
terminal setup (TERM environment variable value)
